### ISSUE TITLE
Code for visulize_input_data_and_network?

### ISSUE THREAD

**Issue by hansaskov** [suyz526/ZebraPose#18]
I'm trying to use my own dataset, and i'm troubleshooting some training problems 

I hacked together a way to visualize the input data as well as the masks, which works okay.

I noticed that the in train.py there is the following uncommented  code.
`
#visulize_input_data_and_network(writer, train_loader, net)
`
Does anyone still have this code? The function doesn't exist in this repo

**Comment by suyz526** [comment]
Hi,

I have some code to visualize the GT image as binary codes.

You can try to modify it. `class_id` refers to the 16 bits binary code in the decimal representation.

```
from class_id_encoder_decoder import class_id_image_to_class_code_images, RGB_image_to_class_image
import numpy as np
import cv2

path_GT_img = "/home/ysu/Downloads/000001_000000.png"

GT_img = cv2.imread(path_GT_img)

class_id_image = RGB_image_to_class_image(GT_img) 
class_code_images = class_id_image_to_class_code_images(class_id_image, 2, 16, 2**16)

for i in range(16):
    img = class_code_images[:,:,i]
    img = img * 256
    filename = "somepath/GT_" + str(i) + ".png"
    cv2.imwrite(filename, img)
```

**Comment by hansaskov** [comment]
Thank you very much. But i cannot import `RGB_image_to_class_image`

It is not available in this repo

**Comment by suyz526** [comment]
Sorry. It has been renamed as `RGB_image_to_class_id_image`.

**Comment by hansaskov** [comment]
Thanks a lot Yongzhi Su

Your visualization works. 

Unfortunately this seems not to be my problem
